Tetris Clone C++/GridController.cpp

```diff
@@ -215,9 +205,28 @@ void GridController::draw() {
             }
         }
     }
+    sf::RectangleShape bbox;
+    sf::Vector2f size = sf::Vector2f((COLUMNS * pixels), (ROWS * pixels));
+    bbox.setSize(size);
+    bbox.setPosition(0,0);
+    bbox.setFillColor(sf::Color(0x000000));
+    bbox.setOutlineColor(sf::Color::White);
+    bbox.setOutlineThickness(1);
+
+    sf::Font font;
```

Comment:
These lines repeatedly declare a font and load from the disc. I suggest changing the font to static or making the font a class member and using dependency injection